src/github.com/getlantern/lantern-ui/app/angular-feed/scripts/services.js

```diff
@@ -37,7 +37,8 @@ angular.module('feeds-services', []).factory('feedService', ['$q', '$http', '$sc
           deferred.reject(new Error("invalid data format"));
           return
         }
-        feedCache.set(feedURL, data);
+        // Always get fresh copy, in hope to be more reactive to network errors
+        // feedCache.set(feedURL, data);
```

Comment:
It's still not unclear to me if we should keep the 5 minutes caching or not. Occasionally it took me around 30 seconds to get a response, even when the response is "304 Not modified". Caching would help for frequent refreshes.